torchao/quantization/quantize_/workflows/intx/intx_opaque_tensor.py

```diff
@@ -15,6 +15,7 @@
 from torchao.quantization.quant_primitives import _DTYPE_TO_BIT_WIDTH
 from torchao.quantization.quantize_.workflows.intx.intx_unpacked_to_int8_tensor import (
     IntxUnpackedToInt8Tensor,
+    ActivationQuantization,
```

Comment:
naming is a bit too general I feel, how about:

```
IntxUnpackedToInt8ActivationQuantization
```

we can move this to `intx` folder and rename to `IntxActivationQuantization` if this enum can be reused by other intx tensors